pyqtgraph/tests/image_testing.py

```diff
@@ -306,15 +309,25 @@ def saveFailedTest(data, expect, filename):
     img[2:2+diff.shape[0], -diff.shape[1]-2:-2] = diff
 
     png = makePng(img)
+    directory, _, _ = filename.rpartition("/")
```

Comment:
FYI this is `os.path.dirname`

Reply:
.. also, not safe to use `"/"` because on windows it's `"\"`; that's what `os.path.sep` is for.

Reply:
you're right, normally I use `pathlib` to get around this... but `os.path.dirname` is the correct call.

Reply:
Fixed with c4e295c